packages/pagebuilder/lib/ContentTypes/Products/products.js

```diff
@@ -95,10 +96,6 @@ const Products = props => {
     const items = restoreSortOrder(skus, data.products.items);
 
     if (appearance === 'carousel') {
-        const galleryItems = items.map((item, index) => {
-            return <GalleryItem key={index} item={item} />;
```

Comment:
`GalleryItem` takes a store config object from the parent, telling it whether to render the wishlist button or not. In this case I did not want to add duplicative logic to this `product.js` file since it potentially already renders a `Gallery` component which contains that logic. Instead I made a new `Carousel` component that contains the same logic as the `useGallery` component and passes the store config to the children.